ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/orchestration.md

```diff
@@ -1,33 +1,74 @@
 # DPP Orchestration Layer with OCI Event and Function: setup guide
 
 ## Scenario
-This doc is to show how to setup OCI Events and Functions to make them work in synergy. In this example, the scenario is once a config file in `json` is uploaded to assigned **object storage bucket**, an **event** listening to the bucket will be triggered and it sends a signal to **OCI Function**, which will read the uploaded json and trigger a **OCI Data Flow** job. We will use the [Remove unnecessary column](./remove_unnecessary_columns.md) as the data flow use case here.
+
+This doc is to show how to setup OCI Events and Functions to make them work in synergy. In this example, the scenario is
+once a config file in `json` is uploaded to assigned **object storage bucket**, an **event** listening to the bucket
+will be triggered and it sends a signal to **OCI Function**, which will read the uploaded json and trigger a **OCI Data
+Flow** job. We will use the [Remove unnecessary column](./remove_unnecessary_columns.md) as the data flow use case here.
```

Comment:
We need to update this to our application rather than remove_unnecessary_columns